docs/concepts/hyperparameters.md

```diff
@@ -1,5 +1,103 @@
+    model_options={}, # Additional options to pass to the model constructor.
+    loss='TripletMarginLoss', # Use CLIPLoss for CLIP fine-tuning.
+    miner='TripletMarginMiner',
+    miner_options={'margin': 0.2}, # Additional options for the miner constructor.
+    scheduler='linear', # Use a linear scheduler to adjust the learning rate.
+    scheduler_options={}, # Additional options for the scheduler.
+    optimizer='Adam',
+    optimizer_options={'weight_decay': 0.01}, # Additional options for the optimizer.
+    learning_rate = 1e-4,
+    epochs=5,
+    batch_size=128,
+    scheduler_step='batch',
+    freeze=False, # If applied will freeze the embedding model, only train the MLP.
+    output_dim=512, # Attach a MLP on top of embedding model.
+    device='cuda',
+    num_workers=4,
+    to_onnx=False,  # If set, please pass `is_onnx` when making inference.
+    csv_options=CSVOptions(),  # Additional options for reading data from a CSV file.
+    public=False,  # If set, anyone has the artifact id can download your fine-tuned model.
+    num_items_per_class=4,  # How many items per class to include in a batch.
+)
+```
+
+Several of these parameters are explained in detail in their own sections,
+the rest of this section will explain how best to use the ones that aren't.
+
+## Learning Rate
+The learning rate affects how much new training data affects the weights of the model you are training,
+a higher learning rate results in a greater change.
+It is important to choose an appropriate learning rate when training;
+too high and you will overfit on your training data and you will lose the knowledge gained from pre-training,
+too low and the model will not chage enough to produce significant improvement.  
+Typically, an apropriate learning rate lies in the range of `1e-4` to `1e-6`. To see a typical learning rate for the different models and tasks we support, check out our [examples](#todo: figure out what to put here).
+
+## Epochs
+Each epoch is a single round of training, in each of these epochs, the model is trained on the entire training
+data, so training for a higher number of epochs will result in a larger change in the model's performance.
+Typically 2-3 epochs are enough for a fine-tuning job.
+
+## Batch Size
+During fine-tuning, your training data is split up into batches and is trained on one batch at a time.
+The `batch_size` parameter can be used to configure the size of each batch.
+A larger `batch_size` results in faster training, though too large a `batch_size` can result
+in out of memory errors. Typically, a `batch_size` of 64 or 128 are good options when you
+are unsure of how high you can set this value, however you can also choose to not set the `batch_size`
+at all, in which case the highest possible value will be calculated for you automatically.
+
+## Learning Rate Scheduler
+You can configure Finetuner to use a learning rate scheduler.
+The scheduler is used to adjust the learning rate during training.
```

Comment:
```suggestion
A learning rate scheduler adjusts the learning rate over the course of training.
This can speed up training under some conditions.
You can configure Finetuner to use a learning rate scheduler by setting the `scheduler` parameter.
```

This would also be an excellent place to say what schedulers are available!